### DISCUSSION TITLE
Improving participatory mapping and critical cartography

### DISCUSSION THREAD

**Discussion by KenNL42** [albertkun/21S-ASIAAM-191A#88] [Past Thinking Caps]
Since maps become more electronically, we can improve maps by including many maps into one to raise awareness of area that people do not realize. Map can be biased as it leaves certain information out. This can cause misunderstanding to readers if they do not see the full picture of what is in the map. As technology progresses, map makers can try to include many maps with different aspects to show readers what are underlying information that they may have missed. One example that I can think of is google maps. The original map shows terrain but it does not indicate heights and environment around an area. Having satellite map can give another perspective of how an area looks like, such as the city is crowded or forest is dense in an area.

**Comment by Wny-Duong** [comment]
In regards to the use of satellite mapping, we also have to consider the ethical cases of how that kind of technology can be used.  It's a very useful tool when it comes to giving people the ability to essentially view the outside world from a single device, but it also has some serious concerns when it comes to its user design and ethics. In regards to privacy of local communities, Google Street View employs an opt-out tactic when it comes to the privacy of the homes and locations that they visit.  This opt-out measurement is done in a very confusing way that is essentially hidden behind the default interface of Google Street View. To actually opt-out, you'd need to know that you would have to click on unclearly-labeled (...) additional settings button on the top-left corner of the screen, report a problem (the wording of which carries some negative implication about your own personal concern about the privacy of your data, but I think I'm nit-picking  by taking it to that extreme), select what personal material you want blurred, specify that personal material of yours to an exact, identifiable detail, and then give Google your e-mail address to continue the blurring process.

By default, that technically means that most of the image data acquired, images of homes, vehicles, or individual persons, is more likely than not, taken without the consent of the owners and people involved. To put so much work into barring a person's ability to scrub their geographic data from the web seems alarming to some individuals. In the best case, this is an unethical violation of a person's privacy with no real tangible harm being done. In the worst case, this kind of reckless attitude towards collecting geographic data could expose or endanger some people by allowing people with the ability to do harm to begin to personally identify people and their belongings/home.

**Comment by avaitt** [comment]
I really like the idea of combining or clustering maps to raise awareness on certain niche issues. I think aggregating data to create more comprehensive maps might help convey the bigger picture in a more succinct way. The example you brought up concerning the use of terrain and how certain metrics can become skewed helped show that maps are often one-dimensional and may need to be supplemented at times to prevent misunderstanding in a user. I personally am a fan of interactive mapping and 3D visualizations where we can see a metric like a terrain as a 3D surface as opposed to a 2D topographic visualization. There is still so much technology has to offer us and I think with more innovation we will be able to convey hundreds of ideas in an aesthetic, yet informative, fashion.
